src/config/dynamic-theme-fixes.config

```diff
@@ -4529,6 +4529,31 @@ CSS
 
 ================================
 
+matomo.org
+
+INVERT
+img[alt*="WordPress"]
+img[src*="Mailfence"]
+img[src*="sampling"]
+img[src*="logo"]
+img.mega-menu-logo
+
+CSS
+select, .elementor-text-editor, .elementor-text-editor *,
+.elementor-blockquote *, .elementor-testimonial *,
+.elementor-price-table * {
+    color: #b3aca2 !important;
```

Comment:
Change it to
`color: var(--darkreader-neutral-text) !important;`

Reply:
Changed:
color: var(--darkreader-neutral-text) !important;